ecs-cli/modules/commands/cluster/cluster_command.go

```diff
@@ -68,7 +68,7 @@ func clusterUpFlags() []cli.Flag {
 		},
 		cli.BoolFlag{
 			Name:  command.CapabilityIAMFlag,
-			Usage: "Acknowledges that this command may create IAM resources.",
+			Usage: "Acknowledges that this command may create IAM resources. Not needed if custom instance role is specified.",
```

Comment:
Hm, but it's not strictly optional, right? I feel like it's different from the other optional flags because it's mutually exclusive with the InstanceRoleFlag.